mlir/lib/Conversion/EmulateFp8ExtTrunc/EmulateFp8ExtTrunc.cpp

```diff
@@ -119,7 +127,18 @@ static Value getFloatValueTableFor(Type elementType, Operation *op,
   const auto &sem = type.getFloatSemantics();
   for (uint32_t i = 0; i < 256; ++i) {
     APFloat entry(sem, APInt(8, i));
-    tableElems.push_back(entry.convertToFloat());
+    float x = entry.convertToFloat();
```

Comment:
Originally I broke this up so I could print the table as it was generated, but then it became handy for the ugly hack.